partition/roles/ztp/README.md

```diff
@@ -4,16 +4,19 @@ Configures a server for providing zero-touch-provisioning scripts for switches.
-| -------------------- | --------- | ----------------------------------------------------------- |
-| ztp_nginx_image_name | yes       | the docker image to use to serve ztp scripts.               |
-| ztp_nginx_image_tag  | yes       | the tag of the docker image to use to serve ztp scripts.    |
-| ztp_host_dir_path    |           | the path to serve ztp scripts from.                         |
-| ztp_listen_address   |           | the address used to serve ztp requests                      |
-| ztp_port             |           | the port to serve ztp scripts on.                           |
-| ztp_authorized_keys  | yes       | the authorized keys that should be installed by ztp.        |
-| ztp_admin_user       |           | the user for which the authorized keys will be provisioned. |
-| ztp_additional_files |           | puts additional files into serve directory.                 |
+| Name                         | Mandatory | Description                                                                                               |
+| ---------------------------- | --------- | --------------------------------------------------------------------------------------------------------- |
+| ztp_nginx_image_name         | yes       | the docker image to use to serve ztp scripts.                                                             |
+| ztp_nginx_image_tag          | yes       | the tag of the docker image to use to serve ztp scripts.                                                  |
+| ztp_host_dir_path            |           | the path to serve ztp scripts from.                                                                       |
+| ztp_listen_address           |           | the address used to serve ztp requests                                                                    |
+| ztp_port                     |           | the port to serve ztp scripts on.                                                                         |
+| ztp_authorized_keys          | yes       | the authorized keys that should be installed by ztp.                                                      |
+| ztp_admin_user               |           | the user for which the authorized keys will be provisioned.                                               |
+| ztp_additional_files         |           | puts additional files into serve directory.                                                               |
+| ztp_sonic_nameservers        |           | the nameservers to put into resolv.conf for sonic                                                         |
+| ztp_sonic_extended_cacl.ipv4 |           | iptables ipv4 rules that should be added as extended Control Plane ACLs (Edgecore Sonic specific feature) |
+| ztp_sonic_extended_cacl.ipv6 |           | iptables ipv6 rules that should be added as extended Control Plane ACLs (Edgecore Sonic specific feature) |
```

Comment:
You need to explain why SONiC variables appear in this role in a paragraph / section at the top of the role. So what the intended purpose is and how it is supposed to be used.

Probably it's also worth to add a `NOTEWORTHY` section if this PR means that the way of how SONiC switches are supposed to be bootstrapped change.